getting-started/openstack/installation/ubuntu.md

```diff
@@ -35,20 +35,7 @@ These steps are detailed in this section.
     ```bash
     add-apt-repository ppa:project-calico/{{ ppa_repo_name }}
     ```
-
-1.  Add the official [BIRD](http://bird.network.cz/){:target="_blank"} PPA. This PPA contains
-    fixes to BIRD that are not yet available in Ubuntu. To add the PPA, run:
-
-    ```bash
-    add-apt-repository ppa:cz.nic-labs/bird
-    ```
-
-    > **Tip**: If the above command fails with error
-    > `'ascii' codec can't decode byte`, try running the command with a
-    > UTF-8 enabled locale:
-    > `LC_ALL=en_US.UTF-8 add-apt-repository ppa:cz.nic-labs/bird`.
-    {: .alert .alert-success}
-
+    
```

Comment:
I'm happy with the logic of this change for Focal, but I think we should keep the text - for now - for people installing on Ubuntu older than Focal.  Can you keep the text but add a condition saying that the PPA is only needed for Ubuntu before Focal?